src/installer/managed/Microsoft.NET.HostModel/Microsoft.NET.HostModel.csproj

```diff
@@ -35,5 +36,12 @@
     <Compile Include="$(CoreClrProjectRoot)tools\Common\System\Collections\Generic\ArrayBuilder.cs" Link="Common\ArrayBuilder.cs" />
   </ItemGroup>
 
+  <ItemGroup>
+    <!-- <ProjectReference
+      Include="..\Microsoft.NET.HostModel.MachO.BinaryFormat.Generator\Microsoft.NET.HostModel.MachO.BinaryFormat.Generator.csproj"
+      PrivateAssets="all"
+      OutputItemType="Analyzer"
+      ReferenceOutputAssembly="false" /> -->
+  </ItemGroup>
```

Comment:
I assume it's a `TODO: cleanup before merging`? ;)

Reply:
Oops, amended the commit and forgot to push before opening the PR. Thanks for pointing that out.